File: H/cpp/code.cpp
#include <vector>
#include <iostream>

using namespace std;

long long getEnergyForUnion(const vector<int>& stones) {
    // your code goes here
    return 0;
}

vector<int> readList(int n) {
    vector<int> res(n);
    for (int i = 0; i < n; i++) {
        cin >> res[i];
    }
    return res;
}

int main() {
    int n;
    cin >> n;
    vector<int> stones = readList(n);

    cout << getEnergyForUnion(stones);
}

File: F/js/code.js
function convertToGoodString(probablyBadString) {
    // your code goes here
    return "";
}

const _readline = require('readline');

const _reader = _readline.createInterface({
    input: process.stdin
});

const _inputLines = [];
let _curLine = 0;

_reader.on('line', line => {
    _inputLines.push(line);
});

process.stdin.on('end', solve);


function solve() {
    const probablyBadString = readLine();
    const ans = convertToGoodString(probablyBadString);
    console.log(ans);
}

function readLine() {
    const line = _inputLines[_curLine];
    _curLine++;
    return line;
}
